src-tauri/src/configs/config_ui.rs

```diff
@@ -53,6 +53,7 @@ pub struct ConfigUIContent {
     sharing_enabled: bool,
     visual_mode: bool,
     show_experimental_settings: bool,
+    warmup_seen: Option<bool>,
```

Comment:
We should probably use bool?

Reply:
i didn't wanna set it because of the old config part but yeah will just set to bool &  false ty:D